src/access_nri_intake/data/metadata_schema_experiment.json

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
I removed the `$id` value from here (and from the other schema) because it just pointed to the GitHub raw location of the file - should this be reinstated?

Reply:
Honestly not sure. Any comment @CodeGat or @jo-basevi?

Reply:
I think it was added initially as setting `$id` to an URL as an identifier was mentioned as best practice in json-schema docs (https://json-schema.org/understanding-json-schema/basics#declaring-a-unique-identifier). I think `$id` is only strictly necessary when referencing a schema from another (https://www.learnjsonschema.com/2020-12/core/id/), which isn't the case here. But maybe if future iterations of the schema use that functionality, it might be worth keeping it in for now? 

 Other thoughts to keep it are that it is consistent with other schemas in the [ACCESS-NRI/schema](https://github.com/ACCESS-NRI/schema) repository, and it self-documents an URL that retrieves the schema?

Reply:
I think keep it for the above reasons.